workload/src/main/proto/database.proto

```diff
@@ -0,0 +1,33 @@
+// Copyright 2021 Google LLC
```

Comment:
Chris mentioned this too earlier, and I pushed this off into #43 for now since I couldn't find a quick solution to point the compiler to use proto files that aren't in the standard maven file structure (in `src/main/proto` of the module). However, I can look into fixing that in this PR if you'd prefer.

Reply:
I was looking into this the other day. So the way to do this would be:
1. Create a new maven module (serviceprotos) which hosts only the service protos. This would have its own pom with dependencies for grpc and protobuf. You can either put the proto file in src/main/proto in this module OR put it in the main folder updating the plugins protoSourceRoot attribute - https://www.xolstice.org/protobuf-maven-plugin/compile-mojo.html#protoSourceRoot
2. To finapp's (parent) pom, we add this serviceprotos module under <modules>
3. To both server and workload poms, we add the dependency:
```
        <dependency>
            <groupId>org.example</groupId>
            <artifactId>serviceprotos</artifactId>
            <version>1.0-SNAPSHOT</version>
        </dependency>
```

Reply:
Ah, I was looking into this protoSourceRoot earlier. I think the plan for now is to just specify the filepath to the server module (and include it as a dependency) and remove the copies. 

I think a greater restructuring should happen in another PR. I'll copy your comment into #43.

Does this sound alright?

Reply:
This change is reflected now in the most recent commit.